etcdserver/etcdhttp/http.go

```diff
@@ -38,7 +38,10 @@ const (
 	defaultWatchTimeout = time.Duration(math.MaxInt64)
 )
 
-var errClosed = errors.New("etcdhttp: client closed connection")
+var (
+	plog      = capnslog.NewPackageLogger("github.com/coreos/etcd", "etcdhttp")
```

Comment:
this is the pkg name. it should be just `etcdhttp`. it is short and clear in my opinion.